Code/Framework/AzToolsFramework/AzToolsFramework/ViewportSelection/EditorVisibleEntityDataCache.h

```diff
@@ -55,7 +55,11 @@ namespace AzToolsFramework
         bool IsVisibleEntityVisible(size_t index) const;
         bool IsVisibleEntitySelected(size_t index) const;
         bool IsVisibleEntityIconHidden(size_t index) const;
-        bool IsVisibleEntitySelectableInViewport(size_t index) const;
+        //! Returns true if the entity is individually selectable (not inside an entity container)
```

Comment:
nit: "not inside an entity container" doesn't seem like quite the right description. I think "its nearest container entity ancestor is being focused" is more accurate but maybe @AMZN-daimini could weigh in.

Reply:
If you pass this an `EntityId` of a 'contained' entity then it won't be technically individually selectable (hence the name change), but I could try and phrase this in an easier to understand way. If you have any suggestions @AMZN-daimini please let me know. Thanks!

Reply:
Container Entity is an Editor concept now, and that's what we're referring to here. (The Prefab Container, incidentally, is also a Container Entity, but this code isn't explicitly aware of Prefabs, and the Container Entity concept can be used in other circumstances in the future).

To make this comment clearer, I would reword it like this:
```
//! Returns true if the entity is individually selectable (none of its ancestors is a closed container entity).
//! @note It may still be desirable to be able to 'click' an entity that is a descendant of a closed container
//! to select the container itself, not the individual entity.
```

Reply:
Comment is now updated